keyboards/dp3000/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
Encoder lines can be removed as they have been specified at keyboard level
```suggestion
```

Reply:
resolved on new commit, and i move encoder at keyboard level from rules.mk to info.json